scripts/device/xcvu13p-fhgb2104-2-i.tcl

```diff
@@ -5,16 +5,16 @@ dict set cfg user_clk 500000000
 dict set cfg sys_clk 100000000
 dict set cfg pcie_clk 100000000
 dict set cfg qsfp_clk 161132812
+dict set cfg qsfp_clk_M 161.1328125
+dict set cfg qsfp_lanes 4x25
 dict set cfg cmac_property [list \
     CONFIG.ADD_GT_CNRL_STS_PORTS {0} \
     CONFIG.CMAC_CAUI4_MODE {1} \
-    CONFIG.CMAC_CORE_SELECT {CMACE4_X0Y7} \
     CONFIG.ENABLE_AXI_INTERFACE {0} \
-    CONFIG.GT_GROUP_SELECT {X0Y32~X0Y35} \
-    CONFIG.GT_REF_CLK_FREQ {161.1328125} \
+    CONFIG.GT_REF_CLK_FREQ $qsfp_clk_M \
```

Comment:
When run your scripts, the following error generated:
```
can't read "qsfp_clk_M": no such variable
    while executing
"list \
    CONFIG.ADD_GT_CNRL_STS_PORTS {0} \
    CONFIG.CMAC_CAUI4_MODE {1} \
    CONFIG.ENABLE_AXI_INTERFACE {0} \
    CONFIG.GT_REF_CLK_FREQ $qsfp_..."
    invoked from within
"dict set cfg cmac_property [list \
    CONFIG.ADD_GT_CNRL_STS_PORTS {0} \
    CONFIG.CMAC_CAUI4_MODE {1} \
    CONFIG.ENABLE_AXI_INTERFACE {0} \
    C..."
    (file "./device/xcvu13p-fhgb2104-2-i.tcl" line 10)

    while executing
"source ./device/${device}.tcl"
    (file "build.tcl" line 48)
```

Please double check your scripts before submit.